website/src/content/docs/docs/migrate-swagger/faq/breakingchange.md

```diff
@@ -150,23 +180,32 @@ You can generally choose an asynchronous operation template that matches your op
   ```tsp
   op doStuff is ArmResourceActionAsync<Resource, ActionRequest, ActionResponse>;
 
-  // with no request body
+  // With no request body
   op doStuffNoRequest is ArmResourceActionAsync<Resource, void, ActionResponse>;
 
-  // with no response body
+  // With no response body
   op doStuffCommand is ArmResourceActionAsync<Resource, ActionRequest, void>;
   ```
 
 #### Templates for Async DELETE Operations
 
-- `ArmResourceDeleteWithoutOKAsync` is a DELETE operation that uses no request body, will return a `202` response in the case of an Asynchronous delete operation, and a `204` response in case the resource does not exist.
+- `ArmResourceDeleteWithoutOKAsync` is a DELETE operation that uses no request body, will return a `202` response in the case of an asynchronous delete operation, and a `204` response in case the resource does not exist.
 
   ```tsp
   op delete is ArmResourceDeleteWithoutOKAsync<Resource>;
   ```
 
-- `ArmResourceDeleteAsync`iis a DELETE operation that uses no request body, and return a `200` response in the case of a successful synchronous delete, a `202` response in the case of an Asynchronous delete operation, and a `204` response in case the resource does not exist.
+- `ArmResourceDeleteAsync` is a DELETE operation that uses no request body, and returns a `200` response in the case of a successful synchronous delete, a `202` response in the case of an asynchronous delete operation, and a `204` response in case the resource does not exist.
 
   ```tsp
-  op createOrUpdate is ArmResourceDeleteAsync<Resource>;
+  op delete is ArmResourceDeleteAsync<Resource>;
   ```
+
+### Visibility Changes for `nextLink` and `value` Properties
```

Comment:
This has no SDK impact as far as I'm aware, since those properties are always internal. Should we accept the Swagger diff here, and remove the entire section?

I'm just worried we ask service teams to do useless work here, there is already enough of important work.

Reply:
Agreed that these changes make no difference to the SDK, or the API in general, because they are response-only types.  If possible, we should look at loosening this rule to exclude response-only types.